files/es/web/api/batterymanager/dischargingtime/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: BatteryManager.dischargingTime
+slug: Web/API/BatteryManager/dischargingTime
+tags:
+  - API
+  - Apps
+  - Batería
+  - Battery
+  - Firefox OS
+  - Gecko DOM Reference
+  - WebAPI
+  - aplicaciones
+translation_of: Web/API/BatteryManager/dischargingTime
+browser-compat: api.BatteryManager.dischargingTime
+---
+
+{{deprecated_header}}{{APIRef("Battery API")}}
+
+Indíca la cantidad de tiempo, en segundos,
+que restan antes de que la batería se descargue completamente.
+
+> **Nota:** Incluso si el tiempo devuelto es exacto al segundo,
+> los navegadores lo redondean a valor más alto.
+> intervalo (típicamente a los 15 minutos más cercanos) por razones de privacidad.
+
+## Sintaxis
+
+```js
+var time = battery.dischargingTime;
+```
+
+Valor de retorno, `time` es el tiempo restante en segundos antes que la `batería`,
```

Comment:
This is maybe more of an opinion, but since `battery` is an object and not just a vocabulary term, we'd usually keep it in English